src/language-js/utils.js

```diff
@@ -1337,6 +1333,18 @@ function getComments(node, flags, fn) {
 const isNextLineEmpty = (node, { originalText }) =>
   isNextLineEmptyAfterIndex(originalText, locEnd(node));
 
+function isCallLikeExpression(node) {
+  return isCallOrOptionalCallExpression(node) || node.type === "NewExpression";
+}
+
+function isCurriedArrowFunctionExpression(node) {
+  return (
+    node &&
```

Comment:
Hi sorry to bother you with another question: do we know if the `node` object reference could ever be missing here and can we think of a way we could test this?

I proposed the simplification here in response to a codecov warning but am now wondering if someone else might think of removing this condition at some point in the future? Or am I getting too fussy here?

Reply:
Yes, it could be `call(, foo)`, it's not valid code, but babel can parse it.

Reply:
If I remove this condition on the `node` object reference, all tests still pass in my work area. When I tried it with `call(, foo)`, it leads to an exception regardless of this condition on the `node` object reference. When I tried merging with PR #10023 and removing this condition, still no test failures. So I am not really convinced at this point.